ops/terraform/services/server/modules/bfd_server_log_alarms/main.tf

```diff
@@ -109,3 +109,31 @@ resource "aws_cloudwatch_metric_alarm" "server-query-logging-listener-warning" {
   datapoints_to_alarm = 1
   treat_missing_data  = "notBreaching"
 }
+
+#BFD-3996 Low serverity notification on occurrance of "SAMHSA 2.0 filter mismatch error"
```

Comment:
No need to annotation your solution in this way–the commit message should be sufficient.
```suggestion
```